slurm/2_ApplicationSpecific/VASP-GPU-podman/sample_SLURM_vasp_std.bash

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -l
 
 # Slurm account
-#SBATCH --account="ccradmin"
+#SBATCH --account="ccrgroup"
```

Comment:
We use `SlurmAccountName` in our docs because this may not be the same as `[YourGroupName]`